test/gleam/bit_array_test.gleam

```diff
@@ -279,3 +279,25 @@ pub fn base16_decode_test() {
   bit_array.base16_decode("a1b2c3d4e5f67891")
   |> should.equal(Ok(<<161, 178, 195, 212, 229, 246, 120, 145>>))
 }
+
+pub fn inspect_test() {
+  bit_array.inspect(<<>>)
+  |> should.equal("<<>>")
+
+  bit_array.inspect(<<80>>)
+  |> should.equal("<<80>>")
+
+  bit_array.inspect(<<0, 20, 0x20, 255>>)
+  |> should.equal("<<0, 20, 32, 255>>")
```

Comment:
Could we have a test for non-byte aligned bit strings please? For example: [this existing one](https://github.com/gleam-lang/stdlib/blob/ef9a6a50bf45ebeba257d2db7aa1116c348d4fb9/test/gleam/string_test.gleam#L1070-L1075)

I believe the implementation doesn't handle this currently.

Reply:
Have added several tests for non-byte aligned bit arrays.